content/english/debugging/04-identify-the-problem3.md

```diff
@@ -9,114 +9,128 @@ draft: false
 
 Debugging code is such a widespread, common problem that people have built entire programs designed to help other programmers debug more efficiently. These are suitably named **debuggers**, and there are plenty of debuggers that work for the C programming language. Let's take a look at `gdb`, a common debugger used with the command line.
 
-[GDB (GNU Project Debugger)](https://sourceware.org/gdb/) is a powerful debugger that lets you debug programs from the command line, which is useful in cases where you don’t have access to a GUI. Open the **Shell** tab in the Replit window below. 
+<iframe height="600px" width="100%" src="https://replit.com/@nuevofoundation/Debugging-Samples-C?lite=true#quicksort/quicksort.c" scrolling="no" frameborder="no" allowtransparency="true" allowfullscreen="true" sandbox="allow-forms allow-pointer-lock allow-popups allow-same-origin allow-scripts allow-modals"></iframe>
 
-First, let's compile the program:
+For our examples, we will make use of the [Quicksort](https://www.youtube.com/watch?v=SLauY6PpjW4) algorithm.

```

Comment:
Use `<a href="..." target="_blank">Quicksort</a>` so that the Quicksort video opens in a new window